cpp/src/lists/copying/segmented_gather.cu

```diff
@@ -13,6 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+#include <cudf/detail/copy_range.cuh>
 #include <cudf/detail/gather.cuh>
 #include <cudf/detail/gather.hpp>
```

Comment:
You may want to remove one, since we may not need both at the same time. Please check.

Reply:
I've removed `<cudf/detail/gather.hpp>`, as per your suggestion.